app/controllers/admin/admins_controller.rb

```diff
@@ -3,7 +3,7 @@ class Admin::AdminsController < ApplicationController
   layout "layouts/admin/application"
 
   def index
-    @receipts = Receipt.status_priority.latest
+    @receipts = Receipt.includes(:user).includes(:room).status_priority.latest
```

Comment:
```suggestion
    @receipts = Receipt.includes(:user, :room).status_priority.latest
```
ri dc hok bé

cơ mà sao chị ko thấy phân trang nhỉ

Reply:
Ơ, ở dưới nè chị

Reply:
rubocop nên em xuống dòng á